src/CMakeLists.txt

```diff
@@ -304,12 +342,20 @@ add_subdirectory(test)
 #
 ## Setup benchmark_hotgym
 #
+# for Windows, link with the static library
+# for Linux,OSx, link with the shared library
+if(MSVC)
+    set(libraryToUse ${core_library})
+else()
+    set(libraryToUse ${src_lib_shared} )
```

Comment:
don't do this, just link below to the lib needed

Reply:
> don't do this, just link below to the lib needed

So, you would rather I put the if(MSVC) around target_link_libraries( )?
Well, ok if you think it is more readable.